dashboard/db/migrate/20230503220558_create_lti_integrations.rb

```diff
@@ -0,0 +1,20 @@
+class CreateLtiIntegrations < ActiveRecord::Migration[6.0]
+  def change
```

Comment:
I think for the soft deletion `paranoia` gem, you would just add `acts_as_paranoid` here on line 2 (like in other models - https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/channel_token.rb#L22 )

I think it's ok either way, but probably easier to start w/ the soft-deletion pattern now.

Reply:
That would be added to the Model, not this migration though (looking at the linked file). Right? Doing a global search, I don't see this being added to a migration anywhere.

Reply:
I agree, seems like we should start with the soft-deletion pattern. I'll add it to the related PR that adds the lti_integrations model.

Reply:
Given the small size of the dataset stored in this table, we can easily add a `deleted_at` column in the future if we want to support soft deletion.